packages/components/src/components/send-email-template/send-email-template.tsx

```diff
@@ -1,34 +1,45 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import { PlatformContext } from '@deriv/shared';
 import Button from '../button/button.jsx';
-import Icon from '../icon/icon.jsx';
+import Icon from '../icon/icon';
 import Text from '../text';
 
+type TSendEmailTemplate = {
+    className?: string;
+    live_chat?: React.ReactNode;
+    lbl_no_receive: string;
+    onClickSendEmail: () => void;
+    resend_timeout?: number;
+    subtitle?: string;
+    title: string;
+    txt_resend: string;
+    txt_resend_in: string;
+};
+
 const SendEmailTemplate = ({
     children,
     className,
+    live_chat,
     lbl_no_receive,
     onClickSendEmail,
     resend_timeout,
     subtitle,
     title,
     txt_resend,
     txt_resend_in,
-    live_chat,
-}) => {
+}: React.PropsWithChildren<TSendEmailTemplate>) => {
     const [is_email_not_received_clicked, setIsEmailNotReceivedClicked] = React.useState(false);
     const [is_resend_btn_disabled, setIsResendBtnDisabled] = React.useState(false);
     const [resend_email_btn_text, setResendEmailBtnText] = React.useState(txt_resend);
-    const { is_appstore } = React.useContext(PlatformContext);
+    const { is_appstore } = React.useContext<{ is_appstore?: boolean }>(PlatformContext);
 
     const timeout_limit = resend_timeout || 60;
-    let resend_interval = null;
+    let resend_interval: number;
 
     React.useEffect(() => {
         return () => {
-            clearInterval(resend_interval);
+            window.clearInterval(resend_interval);
```

Comment:
I'm not sure about these, why we change this? I mean why we ae reading these methods from `window`?

Reply:
anyway it is method of window object. typescript asked me to use this method in this way :)

Reply:
@yauheni-kryzhyk-deriv make sure to add QA checklist in the parent card for this to be tested.